### PR TITLE
Add trademark info; update motto

### PR DESCRIPTION
**Pull request by adamjarling**
## What does this do?

Updates Footer component, and adds Trademark info:

image

### CODE REVIEW

**Comment by heathergreerklein** [comment]
I just confirmed that all our trademarks have gone through! So we can update the text to:
`The Samvera name is a trademark in the USA, Canada, the UK and in the
            European Union. The Samvera logo is a trademark  in the USA, Canada, the UK and in the
            European Union.`

File: components/layout/Footer.jsx [hunk 1]
```diff
@@ -1,32 +1,84 @@
...
+            >
+              <span className="sr-only">{item.name}</span>
+              <item.icon className="w-6 h-6" aria-hidden="true" />
+            </a>
+          ))}
+        </div>
+        <div className="max-w-4xl mx-auto mt-10 text-xs leading-5 text-center text-gray-500">
+          <p className="">&copy; {year} Samvera.org. All rights reserved.</p>
+          <p className="">
+            The Samvera name is a trademark in the USA, the UK and in the
```
Comment (heathergreerklein) [inline components/layout/Footer.jsx:74]: `The Samvera name is a trademark in the USA, Canada, the UK and in the European Union. The Samvera logo is a trademark  in the USA, Canada, the UK and in the European Union.`

**Review by heathergreerklein — CHANGES_REQUESTED** [review]
Updated text because all our trademarks are completed.

File: components/layout/Footer.jsx (hunk 1 above)
Comment (adamjarling) [reply components/layout/Footer.jsx]: @heathergreerklein I updated the text in the latest commit. 👍

**Pull request merged**